packages/paste-core/components/in-page-navigation/src/OverflowButton.tsx

```diff
@@ -46,7 +46,9 @@ export const OverflowButton: React.FC<OverflowButtonProps> = ({
 
   const determineShadow = (): BoxShadow | undefined => {
     if (showShadow) {
-      return isInverse ? theme.shadows.shadowScrollInverse : theme.shadows.shadowScroll;
+      if (position === "left")
+        return isInverse ? theme.shadows.shadowBorderLeftInverse : theme.shadows.shadowBorderLeft;
```

Comment:
question: Should these just be `shadowBorder...` instead of `theme.shadows.shadowBorder...`?

Reply:
we need to import these tokens from `theme.shadows`

Reply:
@serifluous so both way works fine but currently the `shadowBorder... ` is creating some typescript issues